crypto_pnl/console_render.py

```diff
@@ -161,8 +161,8 @@ def render_headers(cls):
     @classmethod
     def render_match(cls, tracker, match):
         buy, sell, fee = match
-        position = Asset(sell.quantity - buy.quantity - fee.quantity, tracker.symbol)
-        position.set_value(sell.value_data - buy.value_data, GAIN_VALUE)
+        position = Asset(sell.quantity - buy.quantity + fee.quantity, tracker.symbol)
```

Comment:
Paying a fee is a disposal. We store regular match events as `(buy, sell, zero)` and fee events as `(buy, zero, fee)` the `fee` is just a `sell`, but tracked separately for clarity.